ui/src/localStorage.js

```diff
@@ -2,9 +2,22 @@ export const loadLocalStorage = () => {
   try {
     const serializedState = localStorage.getItem('state')
 
-    return JSON.parse(serializedState) || {}
+    const state = JSON.parse(serializedState)
+
+    // eslint-disable-next-line no-undef
+    if (state.VERSION !== VERSION) {
+      // eslint-disable-next-line no-console
+      console.log('New version detected. Clearing old settings.')
+      window.localStorage.removeItem('state')
+      return {}
+    }
+
+    delete state.VERSION
```

Comment:
Agreed, this should be made into a reducer, and stored as part of Redux state, instead of living only in localStorage.